### ISSUE TITLE
Macros with optional arguments

### ISSUE THREAD

**Issue by theinze** [tonyhffong/Lint.jl#219]
There seems to be a spurious error for optional arguments in macros ?
```
julia> lintstr("macro foo(x, y) end")
0-element Array{Lint.LintMessage,1}

julia> lintstr("macro foo(x, y=100) end")
1-element Array{Lint.LintMessage,1}:
 none:1 E136 y=100: Lint does not understand macro
```
